src/bus/webSocketConnection.js

```diff
@@ -106,6 +106,97 @@ class WebSocketConnection {
   close() {
     this._webSocket.close();
   }
+
+  /**
+   * Publishes a message to a specific channel on the message bus
+   *
+   * @param {string} serviceClientId Client ID of the message bus service
+   * @param {string} channel Message bus channel the message is being sent to
+   * @param {Object} message Message being sent to the message bus
```

Comment:
Could this actually be Any? I'm thinking it doesn't matter what you send in as long as it is JSON encodeable

Reply:
Yes, you're correct